### DISCUSSION TITLE
Simplify the naming by removing "Any" from names

### DISCUSSION THREAD

**Discussion by tonyarnold** [vapor-community/HTMLKit#84] [Ideas]
I know this was probably a well intentioned change, but I feel like the entire API could be made far simpler by eliding `Any` from the names of the various root protocols. 

Given the changes in Swift 5.7, eventually the API is going to end up with `any AnyElement` and `some AnyElement`, when really it should probably just be `any Element`.

Thoughts?

**Comment by mattesmohr** [comment]
Thats true. And it bothers me too. Even if I caused it a little bit. I'm aware of the changes coming with Swift 5.7 and hope it could help to make the code more readable. 

I want to change a lot with HTMLKit 3.0, but I don't want to promise too much at this point. I just have vision, but it takes time. And at the same time I don't want to throw everything away Mats has written.

Hope for the better xD. 

Anyway I appreciate your thoughts. I am sure I can learn a lot from it and also it nice to see someone uses the package.

**Reply by tonyarnold** [reply]
I don't know that it's about throwing anything Mats has built away - a new major release is a chance to build on top of that work, even if it means changing it (which I'm sure Mats would have done were he still working on the code today). 

Let me know when you're ready to talk through your ideas - I'd love to hear them, and am happy to help.
